src/cdk/commands/aslVisualizationCDKManager.ts

```diff
@@ -0,0 +1,105 @@
+/*!
```

Comment:
`commands/` directory is for vscode commands provided by the extension.

AslVisualizationCDKManager , and many other files in this PR, don't belong in `commands/`. The toplevel `cdk/` or `stepFunctions/` directories are a better place for classes and functions directly related to that functionality, than an unrelated subdirectory.

Same comment for all other files in this PR that are related to general service-related functionality.